internal/apex/stun.go

```diff
@@ -3,114 +3,29 @@ package apex
 import (
 	"fmt"
 	"github.com/libp2p/go-reuseport"
-	"net"
-	"strconv"
-	"time"
-
 	"github.com/pion/stun"
 	"go.uber.org/zap"
+	"net"
 )
 
 const (
+	stunServer  = "stun.l.google.com:19302"
```

Comment:
if we use stunServer1 in utils.go, we can probably get rid of one of the stun server here?